### ISSUE TITLE
Jobs should not retry on bad request

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/sdr-api#208]
e.g.:
```

Dor::Services::Client::UnexpectedResponse: Bad  Request: 400 ({"errors":[{"status":"400","title":"Bad  Request","detail":"Unable to find adminPolicy 'druid:rv981bx1152'"}]})
--
```

**Comment by ndushay** [comment]
I think this was fixed with #319 ??  Is this a dup of #321 ?

**Comment by jcoyne** [comment]
Yeah, this can be considered a duplicate. Or more precisely, that #319 is a duplicate of this.
